prep-javascript-objects/prep-javascript-objects-quiz-notes.md

```diff
@@ -0,0 +1,35 @@
+# prep-javascript-objects-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What data types can object properties hold?
```

Comment:
don't forget the quiz questions

Reply:
I answered the quiz questions, please confirm.